vendor/engines/bulk_email/app/views/bulk_email/bulk_email/job.html.haml

```diff
@@ -0,0 +1,21 @@
+= content_for :tabnav do
+  = render "nav_tab", secondary_tab: "history"
+
+= content_for :h1 do
+  = current_facility
+
+%h2= t("bulk_email.title")
+
+= content_for :sidebar do
+  = render "admin/shared/sidenav_users", sidenav_tab: "bulk_email"
+
+= readonly_form_for :bulk_email_job do |f|
+  = f.input :sender
+  = f.input :created_at
+  = f.input :recipient_list
+  = f.input :start_date
+  = f.input :end_date
+  = f.input :user_types
```

Comment:
And part of me thinks it might be good to group the search criteria and the content/recipient_list.